client/jetpack-cloud/sections/partner-portal/utils.ts

```diff
@@ -167,3 +167,13 @@ export function formatApiPartner( partner: APIPartner ): Partner {
 		} ) ),
 	};
 }
+
+/**
+ * Format the string by removing Jetpack, (, ) from the product name
+ *
+ * @param product Product name
+ * @returns Product title
+ */
+export function getProductTitle( product: string ): string {
+	return product.replace( 'Jetpack ', '' ).replace( '(', '' ).replace( ')', '' );
```

Comment:
```suggestion
	return product.replace( /(?:Jetpack|[)(])/gi, '' );
```

Reply:
I just moved this function from the component to the utils file, so missed this. Thanks for the suggestion. I have updated this.